balboa-client/src/main/scala/com/socrata/metrics/collection/PreBufferQueues.scala

```diff
@@ -1,13 +1,13 @@
 package com.socrata.metrics.collection
 
-import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.{BlockingQueue, LinkedBlockingQueue}
 
 import com.socrata.metrics.components.MetricEntry
 
 trait PreBufferQueue {
-  def queue:SocrataAbstractQueue[MetricEntry]
+  def queue: BlockingQueue[MetricEntry]
```

Comment:
Square wheels are actually completely appropriate in [the right circumstances](https://www.youtube.com/watch?v=LgbWu8zJubo). What we had here was an unnecessary gearbox.